### PR TITLE
Refine project setup instructions

### PR DESCRIPTION
**Pull request by kasperski95**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,14 +48,18 @@ curl -L https://raw.githubusercontent.com/software-mansion/protostar/master/inst
...
+1. Clone this repository
+1. Verify the active Python version: `python -V`
+1. Create Python virtual environment in the project directory: `python -m venv .venv`
+1. Activate environment: `source .venv/bin/activate`
+    - Consider using [direnv](https://direnv.net/) to activate the environment on navigating to the project directory
+1. Upgrade pip: `pip install --upgrade pip`
+1. Install [Poetry](https://python-poetry.org/) — a dependency manager: `pip install poetry`
+1. Install project dependencies: `poetry install`
+    - MacBook M1/M2: `CFLAGS=-I/opt/homebrew/opt/gmp/include LDFLAGS=-L/opt/homebrew/opt/gmp/lib poetry install`
+1.  Verify the setup by running tests: `poe test`
```
Comment (Radinyn) [inline README.md:62]:
```suggestion
1. Install `Python 3.7.12` using the Python version management tool and activate that version
   - To be able to build Protostar, set the following environmental variable before installing Python: `PYTHON_CONFIGURE_OPTS="--enable-shared"`
2. Clone this repository
3. Verify the active Python version: `python -V`
4. Create Python virtual environment in the project directory: `python -m venv .venv`
5. Activate environment: `source .venv/bin/activate`
    - Consider using [direnv](https://direnv.net/) to activate the environment on navigating to the project directory
6. Upgrade pip: `pip install --upgrade pip`
7. Install [Poetry](https://python-poetry.org/) — a dependency manager: `pip install poetry`
8. Install project dependencies: `poetry install`
    - MacBook M1/M2: `CFLAGS=-I/opt/homebrew/opt/gmp/include LDFLAGS=-L/opt/homebrew/opt/gmp/lib poetry install`
9.  Verify the setup by running tests: `poe test`
```

**Review by Radinyn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kasperski95) [reply README.md:62]: I changed the numbers on purpose. Markdown interpreter replaces `1.` with proper numbers.

Comment (mkaput) [inline README.md:62]:
```suggestion
1. Verify the setup by running tests: `poe test`
```

File: README.md [hunk 2]
```diff
@@ -48,14 +48,18 @@ curl -L https://raw.githubusercontent.com/software-mansion/protostar/master/inst
...
-9. Verify the setup by running tests: `poe test`
+1. Install `Python 3.7.12` using the Python version management tool and activate that version
+   - To be able to build Protostar, set the following environmental variable before installing Python: `PYTHON_CONFIGURE_OPTS="--enable-shared"`
+1. Clone this repository
+1. Verify the active Python version: `python -V`
+1. Create Python virtual environment in the project directory: `python -m venv .venv`
+1. Activate environment: `source .venv/bin/activate`
+    - Consider using [direnv](https://direnv.net/) to activate the environment on navigating to the project directory
+1. Upgrade pip: `pip install --upgrade pip`
+1. Install [Poetry](https://python-poetry.org/) — a dependency manager: `pip install poetry`
```
Comment (mkaput) [inline README.md:59]:
```suggestion
1. [Install Poetry](https://python-poetry.org/docs/#installation) — a dependency manager
```

I would rather point to their docs. For example, I am using homebrew for this.

**Review by mkaput — APPROVED** [review]

**Review by mkaput — APPROVED** [review]

**Pull request merged**